config/custom-environment-variables.yaml

```diff
@@ -83,6 +83,11 @@ executor:
                 memory:
                     low: K8S_MEMORY_LOW
                     high: K8S_MEMORY_HIGH
+            # k8s tolerations for approprate pod scheduling.
+            # Value is Array of format { key: 'key', value: 'value', effect: 'NoSchedule'}
+            tolerations:
+              __name: K8S_POD_TOLERATIONS
```

Comment:
`K8S_TOLERATIONS` to be consistent since `K8S_CPU_LOW/HIGH` are also config for pods.

Reply:
`K8S_CPU_LOW/HIGH` was not POD configs, they were args for `hyper-runner`

Reply:
ok 👍  I'm fine with it